lib/membrane_rtc_engine/endpoints/webrtc/track_sender.ex

```diff
@@ -150,11 +183,11 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.TrackSender do
           {[], tracker}
 
         {:status_changed, tracker, :active} ->
-          event = %TrackVariantResumed{}
+          event = %TrackVariantResumed{variant: encoding}
           {[event: {pad, event}], tracker}
 
         {:status_changed, tracker, :inactive} ->
-          event = %TrackVariantPaused{}
+          event = %TrackVariantPaused{variant: encoding}
```

Comment:
I think we should clear the keyframe request here - otherwise, we won't send it once the track is resumed